lld/test/ELF/ppc32-debug-names.s

```diff
@@ -0,0 +1,417 @@
+# ppc-debug-names.s was generated with:
+
+# - clang++ --target=powerpc -g -O0 -gpubnames \
```

Comment:
`-O0` is the default, probably omit it for brevity

Reply:
Done.